server/GameServer.cpp

```diff
@@ -367,52 +367,55 @@ void GameServer::DoTimedUpdates()
 	// Check for timed out game entries
 	for (std::size_t i = gameSessions.size(); i-- > 0; )
 	{
-		// Get the current time difference
 		auto timeDiff = currentTime - gameSessions[i].time;
+		
+		// Check if no updates have occured for a while
+		if (timeDiff < InitialReplyTime)
+		{
+			continue;
+		}
 
 		// Check for no initial update within required time
-		if ((timeDiff >= InitialReplyTime) && ((gameSessions[i].flags & GameSessionReceived) == 0))
+		if ((gameSessions[i].flags & GameSessionReceived) == 0)
 		{
 			LogEndpoint("Dropping Game: No initial Host Info from: ", gameSessions[i].addr.sin_addr.s_addr, gameSessions[i].addr.sin_port);
 
 			// Clear bogus entry
 			FreeGameSession(i);
 			// Update counters
 			counters.numDroppedHostedPokes++;
+			continue;

```

Comment:
Small point, though the diff might have been a little easier to read if the `else` block had been maintained in the initial change, rather than jumping right to using `continue`. I'm not certain, but I suspect a subsequent change to the blocking and indentation to transition to `continue` might have allowed for better tracking and less diff noise.

The change itself looks fine though. Just might have been easier to read as 2 separate commits.